src/app/lib/utilities/moment/index.stories.jsx

```diff
@@ -67,7 +68,6 @@ const OCTOBER = 1571097600000;
 const NOVEMBER = 1573776000000;
 const DECEMBER = 1576368000000;
 
-/* eslint-disable prettier/prettier */
```

Comment:
repost: https://github.com/bbc/simorgh/pull/4178/files#r333233667

Reply:
Misread it first time round, fixed now.